library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/impl/RegistryEntryAttachmentImpl.java

```diff
@@ -98,6 +99,16 @@ public Optional<V> getValue(R entry) {
 		return this.getDefaultValue(entry);
 	}
 
+	@Override
+	public void put(R entry, V value) {
+		RegistryEntryAttachmentHolder.getBuiltin(registry()).putValue(this, entry, value);
```

Comment:
The `registry()` call should have a `this.`, and for consistency with the other code on this file, directly use the property, so:
```suggestion
		RegistryEntryAttachmentHolder.getBuiltin(this.registry).putValue(this, entry, value);
```